wgpu/backends/wgpu_native/_api.py

```diff
@@ -2050,10 +2050,28 @@ def set_bind_group(
         self,
         index,
         bind_group,
-        dynamic_offsets_data,
-        dynamic_offsets_data_start,
-        dynamic_offsets_data_length,
+        dynamic_offsets_data=[],
+        dynamic_offsets_data_start=None,
+        dynamic_offsets_data_length=None,
     ):
+        if (
+            dynamic_offsets_data_start is not None
+            or dynamic_offsets_data_length is not None
+        ):
+            assert (
```

Comment:
Can you please rewrite these assertions to use `if`  and `raise`?

Reply:
I think we also want to check that `dynamic_offsets_data_length >= 0`, because negative indices in Python work, but will produce unexpected results in this case.

Reply:
Updated.